scripts/generate_interfaces

```diff
@@ -2,7 +2,7 @@
 set -euo pipefail
 
 (cd interfaces/ &&
-     npm install --no-package-json &&
+     npm install --no-package-lock &&
```

Comment:
Whaaa??!  Thanks for the fix, it's obviously great.  But why no error message?
```
flutterby:thoth ariels$ npm install --no-way-v6.9.0-of-a-tool-can-be-this-broken-not-even-npm
audited 4391 packages in 5.117s
found 0 vulnerabilities
```

# 🤢